src/components/timeline/index.js

```diff
@@ -22,15 +22,76 @@ type StateProps = {|
 type Props = ConnectedProps<{||}, StateProps, {||}>;
 
 class TimelineImpl extends React.PureComponent<Props> {
+  // This may contain a function that's called whenever we want to remove the
+  // "wheel" listener.
+  _removeWheelListener = null;
```

Comment:
nit: it would be nice to add a type for this indicating that this is a nullable function.